src/skvhttpclient/src/skvhttp/client/SKVClient.cpp

```diff
@@ -108,6 +108,20 @@ boost::future<Response<>> TxnHandle::write(dto::SKVRecord& record, bool erase, d
     return _client->_HTTPClient.POST<dto::WriteRequest>("/api/Write", std::move(request));
 }
 
+boost::future<Response<>> TxnHandle::partialUpdate(dto::SKVRecord& record, std::vector<uint32_t> fieldNamesForUpdate) {
```

Comment:
the vector should be called "fieldsForPartialUpdate" as it contains indices, not names

Reply:
Fixed